src/ui/component/Layout.html

```diff
@@ -53,7 +37,7 @@ <h2>Help us improve Civil Service Learning</h2>
 				</div>
 			</div>
 			<div class="copyright">
-				<a href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/copyright-and-re-use/crown-copyright/">© Crown copyright</a>
```

Comment:
Do we have to support browsers that don't support `utf-8`? 😂

Reply:
#accessible :)